docs/switching-between-main-and-replica.md

```diff
@@ -15,5 +15,7 @@ will switch the connection's state to the main database.
     - `SELECT FOR UPDATE` statement.
     - `UPDATE` statement.
     - `DELETE` statement.
-    - an SQL function call.
     - All calls with transaction isolation level higher than `TRANSACTION_READ_COMMITTED`.
+
+6.  The query will use the main database in case it's an unknown function call. Known read-only functions are [standard
```

Comment:
I need to obtain the list of standard SQL read-only functions. The most up to date SQL standard I have found is SQL-92. The newest standards seem to be protected by a paywall.

Reply:
- [ ] Obtain the list of standard SQL read-only functions

Reply:
I think I'll start with https://www.techonthenet.com/postgresql/functions/index.php. We can add more later.

Reply:
Good call